cypress/e2e/home_page.cy.ts

```diff
@@ -21,4 +21,27 @@ describe("The Home Page", () => {
     cy.get("button").contains("Generate key pair for a new account").click();
     cy.get("h2").contains("Create a new account");
   });
+
+  it("Should get an error message when input an invalid secret key format", () => {
+    cy.get("button").contains("Connect with a secret key").click();
```

Comment:
I think this could be an alias and every cy.get("...") that is repeated. You can save it in the beforeEach or `it` if somehow is not visible